Source/JavaScriptCore/runtime/JSStringInlines.h

```diff
@@ -353,6 +353,12 @@ inline void JSRopeString::resolveToBuffer(JSString* fiber0, JSString* fiber1, JS
                     view0.substring(offset, rope0Length).getCharacters(buffer);
                 } else
                     resolveToBuffer(rope0->fiber0(), rope0->fiber1(), rope0->fiber2(), buffer.first(rope0Length), stackLimit);
+
+                // Both ropes are the same fibers! Then we can just copy the previously generated characters.
```

Comment:
Would it be worth adding this to the 3 fiber case?

Reply:
I collected information, and actually I didn't see any instances. So for now, I'm focusing on two ropes case. If we see this is useful, we will do it in the future.